public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/schedule-grid.js

```diff
@@ -0,0 +1,331 @@
+
+// todo conver all const { to import
+
+/**
+ * WordPress dependencies
+ */
+import { __experimentalGetSettings, format } from '@wordpress/date';
+const { Component, Fragment } = wp.element;
+const { __, _x }              = wp.i18n;
+
+/**
+ * Internal dependencies
+ */
+import { NoContent } from '../../components/';
+import { Sessions } from './sessions';
+
+
+
+
+/* todo
+ *
+ *
+ * IMPORTANT: refer to https://codepen.io/mrwweb/pen/ZaONLW for lots of critical notes and info, also https://meta.trac.wordpress.org/ticket/3117
+ *
+ * markup/css needs to have fallback for unsupported browsers? doesn't have to look good, but has to be legible/accessible
+ *      will babel add prefix for ie11?
```

Comment:
I'd specifically recommend _against_ IE11 grid. They should just receive the fallback/mobile version. Because of that `@supports( display: grid ) {}` is perfect because IE11 "supports" grid but doesn't support `@supports`. Reference: https://css-tricks.com/building-a-conference-schedule-with-css-grid/#article-header-id-2